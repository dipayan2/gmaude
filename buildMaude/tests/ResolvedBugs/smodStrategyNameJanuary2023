#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/smodStrategyNameJanuary2023.maude -no-banner \
  > smodStrategyNameJanuary2023.out 2>&1

diff $srcdir/smodStrategyNameJanuary2023.expected smodStrategyNameJanuary2023.out > /dev/null 2>&1
