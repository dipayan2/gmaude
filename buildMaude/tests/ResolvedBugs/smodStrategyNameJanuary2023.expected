Advisory: Ignoring a specific strat mapping because strategy expand comes from
    a parameter.
Bye.
